sources/packages/web/src/views/institution/locations/active-applications/ActiveApplicationReportAChange.vue

```diff
@@ -30,12 +35,50 @@ export default {
     },
   },
   setup(props: any) {
-    const initialData = ref({});
+    const router = useRouter();
+    const initialData = ref({} as ApplicationDetails);
+    const loadInitialData = async () => {
+      const applicationDetails: ApplicationDetailsForCOEDTO = await ConfirmationOfEnrollmentService.shared.getApplicationForCOE(
```

Comment:
Does this method return applications that are "on COE", right (Required, Completed, Declined)?
My understanding is that the applications listed here must be the ones with the COE completed as per the below AC:
"Student Changes in Circumstances after Confirmation of Enrolment ". Does it make sense?

The method in the summary is using the "_:locationId/active-applications_" endpoint, which means that not every application listed in the summary will be retrieved here.

Reply:
yes, I agree, updated